tests/zenko_tests/node_tests/ui/cypress/integration/bucket_spec.js

```diff
@@ -9,7 +9,7 @@ describe('Bucket', () => {
 
         it('should create a bucket', () => {
             cy.createBucket(bucketName);
-            
+
```

Comment:
extra space?

Reply:
I keep the line break, but I remove the useless white spaces